manifests/i/IObit/DriverBooster9/10.0.0/IObit.DriverBooster9.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: IObit.DriverBooster9

```

```suggestion
PackageIdentifier: IObit.DriverBooster10
```
You'll also need to change `manifests/i/IObit/DriverBooster9` to `manifests/i/IObit/DriverBooster10` and update the file names as well.